scripts/layered.sh

```diff
@@ -24,15 +24,15 @@ export PR_REPO=element-web
 node_modules/matrix-react-sdk/scripts/fetchdep.sh matrix-org matrix-js-sdk
 pushd matrix-js-sdk
 yarn link
-yarn install --pure-lockfile
+yarn install --frozen-lockfile
```

Comment:
what is the difference?

Reply:
The main difference is that `yarn install --frozen-lockfile` prevents any changes to the lockfile or node_modules folder, while `yarn install --pure-lockfile` only installs dependencies based on the lockfile without any updates.